src/java.base/share/classes/java/text/ChoiceFormat.java

```diff
@@ -586,12 +586,18 @@ public Number parse(String text, ParsePosition status) {
         return Double.valueOf(bestNumber);
     }
 
+    /**
+     * @since 23
+     */
```

Comment:
IIUC, this addition will add & inherit the javadoc from `NumberFormat`, which is not the case before. The description for NumberFormat does not fit with ChoiceFormat. Probably that needs to be addressed with a different issue.

Reply:
My review is for the ChoiceFormat class only. So asking for more Reviews for other area
/reviewers 2 reviewer

Reply:
Thanks, I will leave this open for a few more days to let more people from the relevant areas review it.